### DISCUSSION TITLE
第2节课笔记(11班)

### DISCUSSION THREAD

**Discussion by keesh0410** [InternLM/Tutorial#107] [作业]
坚持学习的你真的很酷很酷！

**Comment by horacehht** [comment]
https://blog.csdn.net/m0_50123831/article/details/135422798

**Comment by morning-start** [comment]
https://flowus.cn/morningstar/share/aeb2ae5c-f9cf-43b8-8239-c6125a0d882c
【FlowUs 息流】轻松玩转书生·浦语大模型趣味Demo

**Comment by chenyang126** [comment]
https://blog.csdn.net/weixin_50797351/article/details/135428272?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135428272%22%2C%22source%22%3A%22weixin_50797351%22%7D

**Comment by qianlian-mozi** [comment]
https://blog.csdn.net/m0_54824511/article/details/135427873?spm=1001.2014.3001.5501

**Comment by blackxer** [comment]
https://zhuanlan.zhihu.com/p/676420543

**Comment by Shengshenlan** [comment]
https://github.com/Shengshenlan/tutorial/blob/master/note/2/%E7%AC%AC%E4%BA%8C%E8%8A%82%E8%AF%BE%E7%AC%94%E8%AE%B0.md

**Comment by BarryYin** [comment]
第二节课笔记：https://github.com/BarryYin/internLM/blob/main/%E7%AC%AC%E4%BA%8C%E8%8A%82.md

**Comment by Kongsea** [comment]
第二课作业：https://blog.csdn.net/deeepsea/article/details/135436430

**Comment by lvoooo** [comment]
https://github.com/lvoooo/internLM-learning/blob/main/2.md

**Comment by zhanghui-china** [comment]
学习笔记：
[https://zhuanlan.zhihu.com/p/676369044](url)

**Comment by LichKing-W** [comment]
https://zhuanlan.zhihu.com/p/676491418

**Comment by codefd** [comment]
https://juejin.cn/post/7321049431489396773

**Comment by JustLike1** [comment]
https://blog.csdn.net/qq_45607888/article/details/135439848

**Comment by Vacili-f** [comment]
https://zhuanlan.zhihu.com/p/676541837?

**Comment by hellokid123** [comment]
https://github.com/hellokid123/internllm-notes/blob/main/notes/%E7%AC%AC2%E8%AF%BE.md

**Comment by wwewwt** [comment]
https://github.com/wwewwt/-InternLM/wiki/2%E3%80%81%E6%89%93%E5%8D%A1

**Comment by wuyingxue1230** [comment]
https://zhuanlan.zhihu.com/p/676552277

**Comment by Fulin-Guo** [comment]
https://github.com/wdnmdm/Team_learning/blob/main/InternLM/Chapter2.md

**Comment by dukai289** [comment]
https://github.com/dukai289/Learning/blob/master/Learning_InternLM/2_Demo.md

**Comment by GMYelims** [comment]
https://blog.csdn.net/weixin_42112760/article/details/135451212?spm=1001.2014.3001.5501

**Comment by moyitech** [comment]
https://blog.csdn.net/qq_30650051/article/details/135416872

**Comment by seifer08ms** [comment]
https://zhuanlan.zhihu.com/p/676130541

**Comment by Lyusijie** [comment]
https://blog.csdn.net/weixin_44986362/article/details/135398621?spm=1001.2014.3001.5501

**Comment by godlikeS97** [comment]
https://blog.csdn.net/godlikesun97/article/details/135492703?spm=1001.2014.3001.5502

**Comment by Tangent-90C** [comment]
https://zhuanlan.zhihu.com/p/677058423
